app/services/finance/ecf/mentor/statement_calculator.rb

```diff
@@ -138,6 +129,10 @@ def declaration_types
           self.class.declaration_types
         end
 
+        def participant_declarations
+          statement.participant_declarations.voided.merge!(ParticipantDeclaration.mentor)
```

Comment:
I don't think we should have voided here?

Reply:
fixed this